client/extensions/woocommerce/app/order/order-details/table.js

```diff
@@ -74,24 +88,69 @@ class OrderDetailsTable extends Component {
 		);
 	};
 
-	renderOrderItems = item => {
-		const { order, site } = this.props;
+	onChange = event => {
+		const { order } = this.props;
+		// Name is `quantity-x`, where x is the ID of the item
+		const id = parseInt( event.target.name.split( '-' )[ 1 ] );
+		const item = find( order.line_items, { id } );
+		if ( ! item ) {
+			return;
+		}
+		const index = findIndex( order.line_items, { id } );
+		// A zero quantity does strange things with the price, so we'll force 1
```

Comment:
Since we don't allow them to reduce quantity to zero - what's the planned flow for allowing merchants to delete lines? Will we do something (i.e. in a later PR) like core does today and have a "X" that allows for deleting a line while editing an order?

<img width="683" alt="screen shot 2017-10-23 at 11 04 45 am" src="https://user-images.githubusercontent.com/1595739/31905133-0467b6fa-b7e2-11e7-9c7c-d9e0cdbfa750.png">

Reply:
yep #19056 🙂